docs/changelog.md

```diff
@@ -30,7 +30,15 @@ permalink: /changelog/
   }
   ```
 * **feature-prompts**
-  * Added support alerts dialogs.
+  * Added support for alerts dialogs.
+  * Added support for date picker dialogs.
+
+* **support-ktx**
+  New extension function `toDate` that converts a string to a Date object from a formatter input.
+  ```Kotlin
+       val date1 = "2019-11-28".toDate("yyyy-MM-dd")
+       val date1 = "".toDate("yyyy-MM-dd")) // Returns Date()
```

Comment:
Not sure we need to show the second case as a feature i.e. when would you ever do this? :)